### ISSUE TITLE
Playlists don't load

### ISSUE THREAD

**Issue by CharlotteCross1998** [yuliskov/SmartTube#3556]
Title.

Everything else seems to work just fine. On the latest version.

**Comment by dontwantonebutneedone** [comment]
Are you talking about time stamps? If so, same here. Went away this most recent release. I had that problem a while ago, its been gone for months and just came back. Not all of them, just some. Seems if there's any text in there like a url to a source, that gives it trouble.

**Comment by keiron26** [comment]
Yeah I have same issue where playlists just will not load just get loading icon and thats it

**Comment by CharlotteCross1998** [comment]
> Are you talking about time stamps? If so, same here. Went away this most recent release. I had that problem a while ago, its been gone for months and just came back. Not all of them, just some. Seems if there's any text in there like a url to a source, that gives it trouble.

No, I have the issue @keiron26 is describing
